rpms/generate_wazuh_rpm.sh

```diff
@@ -145,6 +145,7 @@ help() {
     echo "    -r, --release             [Optional] Package release. By default: 1."
     echo "    -p, --path                [Optional] Installation path for the package. By default: /var."
     echo "    -d, --debug               [Optional] Build the binaries with debug symbols. By default: no."
+    echo "    -k, --checksum            [Optional] Generate checksum"
```

Comment:
The code to support `-k` option is missing in this script and in `generate_wazuh_debs.sh`.

Reply:
done in https://github.com/wazuh/wazuh-packages/pull/192/commits/c1caa5e45b8d2b16abc8021bcdead50b47f1447f